libbeat/outputs/elasticsearch/config.go

```diff
@@ -15,7 +15,7 @@ type elasticsearchConfig struct {
 	ProxyURL         string             `config:"proxy_url"`
 	LoadBalance      bool               `config:"loadbalance"`
 	CompressionLevel int                `config:"compression_level" validate:"min=0, max=9"`
-	TLS              *outputs.TLSConfig `config:"tls"`
+	TLS              *outputs.TLSConfig `config:"ssl"`
```

Comment:
Not required in this PR but we should also rename it internally to SSL for consistency.